system-addon/content-src/components/Base/_Base.scss

```diff
@@ -93,19 +99,17 @@ main {
     background-color: transparent;
     border: 0;
     cursor: pointer;
-    fill: $fill-secondary;
+    fill: var(--newtab-icon-primary-color);
     offset-inline-end: 15px;
     padding: 15px;
     position: fixed;
     top: 15px;
     z-index: 12001;
 
-    &:hover {
-      background-color: $background-secondary;
-    }
-
+    &:hover,
+    &:focus,
     &:active {
```

Comment:
currently, when you click the background goes away. I guess we can keep that behavior. I dont think it really makes a difference either way since it's behavior is more like a link than a button.